### PR TITLE
Add PiGro

### PR DESCRIPTION
**Pull request by actionschnitzel**
# Please make sure all below checkboxes have been checked before submitting your PR.

- [x] I have read and understood the [contribution guidelines](https://github.com/thibmaek/awesome-raspberrypi/blob/master/CONTRIBUTING.md).
- [x] This pull request has a descriptive title. *(For example: `Add Raspbian`)*
- The topic I added
  - [x] includes a valid (https) link,
  - [x] includes a concise and on-topic description,
  - [x] mentions if there is only support some devices,
  - [x] is not a duplicate,
  - [x] has been in the correct alphabetical order for its section,
  - [x] is in the form of **Named Link - Description, separated by commas.**
  - [x] ends with a dot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ The complete collection of (consumer) Raspberry Pi models consist of:
 - [PiShrink](https://github.com/Drewsif/PiShrink/) - Bash script that automatically shrinks a pi image that will then resize to the max size of the SD card on boot.
 - [pistrong](https://github.com/gitbls/pistrong) - Manage certificate-secured strongSwan VPNs. Installer and command-line tool for client-server, site-to-site, and host-to-host VPNs.
 - [PiVPN](https://pivpn.io) - Create a full-featured OpenVPN/WireGuard VPN Server with a single bash command.
+- [PiGro - Just Click It!](https://github.com/actionschnitzel/PiGro-Aid-) - Everything you need to use the Pi as a desktop computer. PiGro does many commands that have to be entered via the terminal with one or two clicks of a button.
```
[the remark this answers is not in the document]
Comment (thibmaek) [reply README.md:123]: Description itself needs to be one sentence

**Review by thibmaek — CHANGES_REQUESTED** [review]

**Comment by thibmaek** [comment]
Hi! Thanks for the contribution, can you double check the contribution guidelines, the PR checklist and the remarks on this PR in order to complete it?

**Review by actionschnitzel — COMMENTED** [review]
Application name and description shortened.
The full name is "PiGro - Just Click It!"

**Review by thibmaek — APPROVED** [review]

**Comment by thibmaek** [comment]
🤝 Thanks for contributing this resource!

**Pull request merged**